templates/app/commercial-vendor-program.phtml

```diff
@@ -0,0 +1,78 @@
+<?php
+const CVP_CONTACT_EMAIL = 'commercial-vendor-program@getlaminas.org';
+$this->layout('layout::default', ['title' => 'Commercial Vendor Program']);
+?>
+
+<div class="container">
+    <div class="row mt-5">
+        <div class="col">
+            <h1>Commercial Vendor Program</h1>
+        </div>
+    </div>
+    <div class="row align-items-start mt-5">
+        <div class="col-12">
+            <h2>Existing Members</h2>
+        </div>
+    </div>
+
+    <div class="card-deck">
+    <?php foreach($commercialVendors as $commercialVendor): ?>
+        <div class="card">
+            <div class="card-body">
+                <h5 class="card-title"><a target="_blank" href="<?= $this->escapeHtmlAttr($commercialVendor['url']) ?>"><?= $this->escapeHtml($commercialVendor['name']) ?></a></h5>
+                <p class="card-text"><?= $this->escapeHtml($commercialVendor['text']) ?></p>
+            </div>
+            <img class="card-img-bottom px-5 pb-2" src="<?= $this->escapeHtmlAttr($commercialVendor['logo']) ?>" alt="<?= $this->escapeHtmlAttr($commercialVendor['name']) ?>">
+            <div class="card-footer">
+                <a class="card-link" target="_blank" href="<?= $this->escapeHtmlAttr($commercialVendor['url']) ?>">Visit <?= $this->escapeHtml($commercialVendor['name']) ?></a>
+            </div>
+        </div>
+    <?php endforeach ?>
+    </div>
+
+    <h2 class="mt-5">Benefits</h2>
+    <p>
+        When you join the Commercial Vendor Program, you will be recognised as one of the key supporters of
```

Comment:
Any reason for the line breaks?

Reply:
No, other than readability when I'm editing. :smile: